City-of-Helsinki/parking-permits-admin-ui
# Remove crossed out discount prices from UI

## Description

Removes "original" discount prices from the UI and added price formatting fixes in permit summary and edit pages.

In addition some tidying up of the UI, particularly the footer in permit edit and summary views.

## Context

[PV-631](https://helsinkisolutionoffice.atlassian.net/browse/PV-631)

## How Has This Been Tested?

Tested manually. Note that in the summary view, it was not possible to view a complete change list locally, so this should be tested by the reviewer if possible.

## Manual Testing Instructions for Reviewers

See above:  select an existing editable permit, make editing changes and look at any price changes.

[PV-631]: https://helsinkisolutionoffice.atlassian.net/browse/PV-631?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ


![Screenshot from 2023-09-21 12-40-19](https://github.com/City-of-Helsinki/parking-permits-admin-ui/assets/131681805/b14ea845-d572-4190-b176-b687185ce451)
